cloud/pkg/router/constants/constants.go

```diff
@@ -5,4 +5,6 @@ const (
 	RestEndpoint     string = "rest"
 	EventbusProvider string = "eventbus"
 	EventbusEndpoint string = "eventbus"
+	GroupResource    string = "resource"
+	MaxmMessageBytes int64  = 1.2e+7
```

Comment:
Why 12mb?

Reply:
It is just the preliminary conclusion after our group discussion. It could be modified if needed.

Reply:
Better to support config it.

Reply:
I have modified to support config.